website/src/docs/guides/backups.md

```diff
@@ -54,16 +71,39 @@ This way, you can recover from catastrophic failures.
 ![Automatic Backupse](/docs/guides/backups/automatic_backups.dark.webp#dark =414x402)
 :::
 
-## Forks Information
+### Sync Backups with External Cloud Services
+
+Cross Device Sync in Tachiyomi is not planned in the future, but users can use 
+[Autosync for Google Drive](https://play.google.com/store/apps/details?id=com.ttxapps.drivesync)
+in order to sync backup files to Google Drive automatically with the following steps:
+
+1. Install the app from the link above
+
+2. Enable [Automatic Backups](/docs/guides/backups#enable-automatic-backups) and set it to your desired frequency and storage location.
+
+3. Download the latest backup from Google Drive and restore to whatever device you have
+
+Users who are familiar with [FolderSync](https://play.google.com/store/apps/details?id=dk.tacit.android.foldersync.lite)
+or [Tasker](https://play.google.com/store/apps/details?id=net.dinglisch.android.taskerm) can setup auto sync of their backups similarly
+
+## Additional Information for Forks
+
+::: warning
+This section explores some extra details regarding [Official Forks](/forks/).
+:::
+
+All [Official Tachiyomi Forks](/forks/) support the `proto.gz` format to backup/restore your library
```

Comment:
```suggestion
All [Official Tachiyomi Forks](/forks/) support the `.proto.gz` format to backup/restore your library
```
You refer to it as `.proto.gz` elsewhere, so for uniformity

Reply:
Thanks